packages/core/src/browser/view-container.ts

```diff
@@ -673,6 +684,8 @@ export class ViewContainerPart extends BaseWidget {
     readonly onTitleChanged = this.onTitleChangedEmitter.event;
     protected readonly onDidFocusEmitter = new Emitter<this>();
     readonly onDidFocus = this.onDidFocusEmitter.event;
+    protected readonly onDidSubHeadingChangedEmitter = new Emitter<void>();
+    readonly onDidSubHeadingChanged = this.onDidSubHeadingChangedEmitter.event;
```

Comment:
I think `onDidChangeSubHeading(Emitter)` would conform more closely to the guidelines on [event naming](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#event_names).